control-plane-operator/controllers/hostedcontrolplane/kas/params.go

```diff
@@ -293,6 +294,16 @@ func NewKubeAPIServerParams(ctx context.Context, hcp *hyperv1.HostedControlPlane
 	}
 	params.KubeConfigRef = hcp.Spec.KubeConfig
 	params.OwnerRef = config.OwnerRefFrom(hcp)
+
+	if explicitNonRootSecurityContext {
```

Comment:
can we name this the same than https://github.com/openshift/hypershift/pull/777/files#diff-3d4ffda0834e09a59a4d510523932e20b9234bcb9816e40258a0fb32a87a7006R119 i.e `setSecurityContext`, they are the same semantic.

Reply:
Same for any other func with this param pass through the signature